### PR TITLE
fix(dropdown): fix bug where dropdown is focused on initial render

### PR DESCRIPTION
**Pull request by mario-subo**

### CODE REVIEW

File: lib/src/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -70,10 +70,13 @@ export const Dropdown: React.FC<DropdownProps> = React.forwardRef(
         const menuRef = React.useRef<HTMLUListElement>();
         const searchRef = React.useRef<HTMLInputElement>();
 
+        const [prestine, setPrestine] = React.useState<boolean>(true);
+
         const isMobile: boolean = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(safeWindow?.navigator?.userAgent);
 
         const changeValue = React.useCallback(
             (value: string) => {
+                console.log(value);
```
Comment (eweseong) [inline lib/src/Dropdown/Dropdown.tsx:79]: let's remove console

Comment (mario-subo) [reply lib/src/Dropdown/Dropdown.tsx:79]: whoops haha well spotted

File: lib/src/Dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -350,10 +353,10 @@ export const Dropdown: React.FC<DropdownProps> = React.forwardRef(
...
         }, [props.value, props.placeholder, selectedLabel]);
 
-        React.useEffect(() => {
-            // scroll the focused item into view
-            getOptionsRef()?.[focusedIndex]?.scrollIntoView(false);
-        }, [focusedIndex, getOptionsRef]);
+        // React.useEffect(() => {
+        //     // scroll the focused item into view
+        //     getOptionsRef()?.[focusedIndex]?.scrollIntoView(false);
+        // }, [focusedIndex, getOptionsRef]);
```
Comment (kherP) [inline lib/src/Dropdown/Dropdown.tsx:359]: remove commented code?

Comment (eweseong) [reply lib/src/Dropdown/Dropdown.tsx:359]: i believe this code is needed? it helps to scroll the focused item into view

Comment (mario-subo) [reply lib/src/Dropdown/Dropdown.tsx:359]: omg that's not supposed to be commented out lol I committed the wrong thing let me fix it

**Review by eweseong — APPROVED** [review]

**Pull request merged**